docs/aclanthology-index.md

```diff
@@ -0,0 +1,28 @@
+# Index Setup
```

Comment:
Rename `aclanthology.md`? As we figure out search, etc. we'll add to it also.

Reply:
Changed.